### DISCUSSION TITLE
Welcome to djs-moderation-system Discussions!

### DISCUSSION THREAD

**Discussion by LunarTaku** [LunarTaku/djs-moderation-system#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Habeeb30** [comment]
I'm timeouted but in log its coming kicked kindly fix it

**Reply by LunarTaku** [reply]
the issue is fixed thank you for reporting it :)
